### PR TITLE
Fixed configs. Currently working locally.

### PR DESCRIPTION
**Pull request by getvictor**
Did some cleanup of the configs. These configs currently work with my local implementation.

### CODE REVIEW

File: teams/workstations.yml [hunk 1]
```diff
@@ -26,21 +27,15 @@ controls:
...
+  - path: ../lib/macos-device-health.policies.yml
+  - path: ../lib/windows-device-health.policies.yml
+agent_options:
+  path: ../lib/agent-options.yml
 team_settings:
-  webhook_settings:
-    failing_policies_webhook:
-        enable_failing_policies_webhook: true
-        destination_url: https://example.tines.com/webhook
-        policy_ids: [1, 2, 3, 4, 5,6 ,7, 8, 9]
```
Comment (noahtalerman) [inline teams/workstations.yml:43]: Are we pulling these out to simplify the example? Or, did we cut `webhook_settings` functionality?

Comment (getvictor) [reply teams/workstations.yml:43]: Our current `fleetctl apply` does not support team webhook_settings or integrations, right? And I'm using the existing APIs, so `fleetctl gitops` doesn't support these either without additional work.

Comment (noahtalerman) [reply teams/workstations.yml:43]: > `fleetctl apply` does not support team `webhook_settings` or `integrations`, right?

@getvictor I'm not positive.

@mna when you get the chance, can you please sanity check us here?

If this is the case, it makes sense to punt on adding support for `webhook_settings` and `integrations`. We can add support for this in `fleetctl gitops` later.

Comment (mna) [reply teams/workstations.yml:43]: AFAICT that's correct, no webhook or integrations can be set via apply team specs: https://github.com/fleetdm/fleet/blob/main/server/fleet/teams.go#L387

File: default.yml [hunk 2]
```diff
@@ -5,17 +5,19 @@
...
   windows_settings:
     custom_settings:
-    - path: ../lib/windows-screenlock.xml
+    - path: ./lib/windows-screenlock.xml
   scripts:
-  - path: ../lib/collect-fleetd-logs.sh
+    - path: ./lib/collect-fleetd-logs.sh
 queries:
-- path: ./lib/collect-fleetd-update-channel.queries.yml
+- path: ./lib/collect-fleetd-update-channels.queries.yml
```
Comment (noahtalerman) [inline default.yml:17]: Do we need to to add an indent here before each query? (`- path`) I see you added indents for `custom_settings`.

File: teams/workstations-canary.yml [hunk 3]
```diff
@@ -36,12 +37,11 @@ queries:
...
   observer_can_run: true
   automations_enabled: false
-policies: 
-- path: ../lib/macos-device-health.policies.yml
-- path: ../lib/windows-device-health.policies.yml
-- path: ../lib/windows-device-health.policies.yml
-agent_options: ./lib/agent-options.yml
+policies:
+  - path: ../lib/macos-device-health.policies.yml
+  - path: ../lib/windows-device-health.policies.yml
```
Comment (noahtalerman) [inline teams/workstations-canary.yml:42]: Similar comment about indentation here. Just want to make sure we're being consistent w/ other options that accept arrays (`custom_settings`, `scripts`, etc.)

**Review by noahtalerman — APPROVED** [review]
Dropped some comments about indentations. 

Otherwise, looks good to me!

**Comment by getvictor** [comment]
> Dropped some comments about indentations.
> 
> Otherwise, looks good to me!

I'll review the indentation before submitting the next PR.

**Pull request merged**